bower.json

```diff
@@ -17,6 +17,6 @@
   "dependencies": {
     "purescript-datetime": "master",
     "purescript-enums": "master",
-    "purescript-nullable": "master"
+    "purescript-nullable": "main"
```

Comment:
Oh, my.

Reply:
At this point I think we can delete the `master` branch from all those repositories and just have the `main` branch. I _believe_ GitHub redirects 'master' to the default branch automatically (but I may be mistaken -- not able to find a source at the moment).

Reply:
Yeah.....